.github/workflows/android_alarm_manager_plus.yaml

```diff
@@ -26,3 +26,30 @@ jobs:
         run: ./.github/workflows/scripts/install-tools.sh
       - name: "Build Example"
         run: ./.github/workflows/scripts/build-examples.sh android ./lib/main.dart
+
+  android_integration_test:
+    runs-on: macos-latest
+    timeout-minutes: 30
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v2
+      - name: "Install Flutter"
+        run: ./.github/workflows/scripts/install-flutter.sh stable
+      - name: "Install Tools"
+        run: ./.github/workflows/scripts/install-tools.sh
+
+      # Required for Android builds
```

Comment:
This is not required as MacOS latest already has Java 11 pre-installed: https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#java

Reply:
Good find. Probably need to be cleaned up from the other integration jobs as well.

Reply:
Thanks for that! This has been removed